### PR TITLE
Create a news section for multiple sources of information

### PR DESCRIPTION
**Pull request by DataDalton**
Merges the existing current investigation status into a news section and adds current Curseforge news

### CODE REVIEW

**Review by xyzeva — CHANGES_REQUESTED** [review]
Probably change this to a new file (`docs/news.md`) or `docs/users.md`

**Comment by DataDalton** [comment]
I think a news doc would be good but since the readme is the front page of the repo it is important for up-to-date news to be seen first and the timeline can keep old news into it rather than have a separate doc

File: README.md [hunk 1]
```diff
@@ -37,6 +40,21 @@ malware and what it does.
...
+
+Curseforge is fully aware of this ongoing problem and peforming their own investigation. 
+You can see their latest updates in their [Discord](https://discord.com/invite/curseforge) server.
+
+Their most recent announcement was at `1686153780` or Wednesday June 07 2023 16:03:00 GMT+0000, and is as follows follows:
+> 📢 UPDATE: Repeated Questions
+> 
+> Hey all, as we are still seeing some misconceptions regarding the current situations, please note the following:
+> 
+> ONLY Minecraft is affected. This malware specifically targets Minecraft JAR files and nothing else. Any concerned players can rest assured their Addons, CC and non Minecraft mods are safe.
```
Comment (Janmm14) [inline README.md:53]: According to https://github.com/fractureiser-investigation/fractureiser/pull/26#discussion_r1222184746 this statement is wrong!

**Review by Janmm14 — CHANGES_REQUESTED** [review]

**Comment by DataDalton** [comment]
You are correct in that statement being wrong, however, that is their most recent news update, I will add a disclaimer to it with the correct information

**Comment by unascribed** [comment]
The README needs to be short to act as a quick jumping off point for new readers. Adding Curseforge-specific clutter doesn't seem constructive. As far as separate documents, this purpose is already served by the timeline doc.

**Pull request closed without merging**